opam-packages-conversion/bin/config.py

```diff
@@ -28,6 +28,7 @@
     "conf-libev",
     "lablgtk",
     "ssl",
+    "mirage-xen",
```

Comment:
Some of its deps don't convert right now (iirc it fails on assert — `|` in deps).